src/raftstore/store/util.rs

```diff
@@ -534,6 +583,85 @@ impl fmt::Debug for Lease {
     }
 }
 
+/// A remote lease, it can only be derived by `Lease`. It will be sent
+/// to the local read thread, so name it remote. If Lease expires, the remote must
+/// expire too.
+#[derive(Debug)]
+pub struct RemoteLease {
```

Comment:
You can derive `Clone` here so yo just need to call clone on https://github.com/pingcap/tikv/pull/2639/files#diff-eb3473ce982ede05dd987c8036237209R565

Reply:
There is only one function needs to clone `RemoteLease`, and I dont it clonable.